packages/documentation/src/pages/getting-started/internal-tools.mdx

```diff
@@ -1,153 +1,134 @@
+cd devops
+```
 
-The `~/.ssh/config` file on your local system contains configuration to access the SOCKS proxy from the VA network (https://github.com/department-of-veterans-affairs/devops/blob/master/ssh/config#L34).
+7. Checkout a branch using your lowercase first and last name
+```bash
+git checkout -b add-key-first-last
+```
+8. Copy your public key into the `authorized_keys` folder renaming the file with your email address
+```bash
+cp ~/.ssh/id_rsa_vagov.pub ./ansible/roles/dsva-config/files/authorized_keys/YOUR@EMAIL.COM
+```
+9. Edit the `./ansible/global/config.yml` in the `devops` repo by adding your email address to the list of emails under the appropriate group (`developers:`, `devops:`, etc.)
 
-1. Run `ssh socks-va -D 2001 -N &`
-   _Note: the first time you connect to the jumpbox, ssh will prompt to ask
-   if you are sure you want to connect to a new host. You will be unable to
-   respond "yes" if ssh is in the background, so either bring it to the
-   foreground with `fg` or omit the `&` character from the above command._
+*Please try to keep this list alphebetized.*
 
-1. Follow the instructions below to configure your browser.
+10. Commit and push your changes to the `devops` repo
+```bash
+git add .
+git commit -m 'add ssh keys for first-name last-name'
+git push origin add-key-first-last
+```
 
-### Accessing SOCKS proxy from the internet
+11. Create a PR for this branch on Github and comment a link to the PR in the #vetsgov-devops channel on the DSVA Slack indicating that you need your ssh keys added to the `devops` repo.
 
-The `~/.ssh/config` file on your local system contains configuration to access the SOCKS proxy from outside the VA network (https://github.com/department-of-veterans-affairs/devops/blob/master/ssh/config#L28).
+12. Copy the SSH config from the `devops` repo to `~/.ssh`
+```bash
+cp ./ssh/config ~/.ssh/config
+```
 
-1. Run `ssh socks -D 2001 -N &`
-   _Note: the first time you connect to the jumpbox, ssh will prompt to ask
-   if you are sure you want to connect to a new host. You will be unable to
-   respond "yes" if ssh is in the background, so either bring it to the
-   foreground with `fg` or omit the `&` character from the above command._
+13. Add your SSH key to your local agent
+```bash
+ssh-add -K ~/.ssh/id_rsa_vagov
+```
+*If you're on OSX you can configure the ssh client to automatically add your keys with the by adding the following to the top of `~/.ssh/config`*
+```
+  Host *
+    AddKeysToAgent yes
+    UseKeychain yes
+    IdentityFile ~/.ssh/id_rsa
+    IdentityFile ~/.ssh/id_rsa_vagov
+```
 
-1. Follow the instructions below to configure your browser.
+14. Wait for `devops` team to review, approve, and merge your pull request
+
+*Once your pull request is merged, an update job should automatically run to update the jumpboxes and SOCKS proxy endpoints. This job takes about 5 minutes to run. Wait five minutes, then continue with* **Starting the SOCKS proxy**.
```

Comment:
I think I prefer this "continue with" approach to the continuous numbering approach I used above- especially since the instructions branch a little.